flow-preferences/src/main/java/com/tfcporciuncula/flow/BooleanPreference.kt

```diff
@@ -1,21 +1,22 @@
 package com.tfcporciuncula.flow
 
 import android.content.SharedPreferences
-import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.withContext
+import kotlin.coroutines.CoroutineContext
 
 class BooleanPreference(
   keyFlow: Flow<String>,
   private val sharedPreferences: SharedPreferences,
   private val key: String,
-  private val defaultValue: Boolean
-) : Preference<Boolean>(keyFlow, sharedPreferences, key) {
+  private val defaultValue: Boolean,
+  private val context: CoroutineContext
```

Comment:
🔍 In the Android world, `context` already has this strong meaning and I'm a bit concerned that it might create some confusion here. I'm thinking it might be better if we make it more explicit, so renaming it from `context` to `coroutineContext`.

Reply:
Excellent point! 

Updated here https://github.com/tfcporciuncula/flow-preferences/pull/3/commits/fea79202526845dd87345dda14f2b3447587e35a